docs/development/python/task-queue-celery-rabbitmq.md

```diff
@@ -0,0 +1,363 @@
+
+[Install]
+WantedBy=multi-user.target
+    {{< /file >}}
+
+2.  Create a`/etc/default/celeryd` configuration file:
+
+    {{< file "/etc/default/celeryd" py >}}
+# The names of the workers. This example create two workers
+CELERYD_NODES="worker1 worker2"
+
+# The name of the Celery App, should be the same as the python file
+# where the Celery tasks are defined
+CELERY_APP="downloaderApp"
+
+# Log and PID directories
+CELERYD_LOG_FILE="/var/log/celery/%n%I.log"
+CELERYD_PID_FILE="/var/run/celery/%n.pid"
+
+# Log level
+CELERYD_LOG_LEVEL=INFO
+
+# Path to celery binary, that is in your virtual environment
+CELERY_BIN=/home/celery/miniconda3/bin/celery
+    {{< /file >}}
+
+3.  Create log and pid directories:
+
+        sudo mkdir /var/log/celery /var/run/celery
+        sudo chown celery:celery /var/log/celery /var/run/celery
+
+4.  Reload systemctl daemon. You should run this command each time you change the service definition file.
+
+        sudo systemctl daemon-reload
+
+5.  Enable the service to startup at boot:
+
+        sudo systemctl enable celeryd
+
+6.  Start the service
+
+        sudo systemctl start celeryd
+
+7. Check that your workers are running via log files:
+
+        cat /var/log/celery/worker1.log
+        cat /var/log/celery/worker2.log
+
+8. Send some tasks to both workers, in a python shell from the directory `/home/celery/downloaderApp`:
+
+        from downloaderApp import download,list
+        r1 = download.delay('https://www.linode.com/media/images/logos/standard/light/linode-logo_standard_light_large.png', 'linode-logo.png')
+        r2 = list.delay()
+        r2.get(timeout=1)
+
+    Depending on how quickly you enter the commands, the worker for `list` task may finish before the worker for `download` task and you may not see the linode logo in the list. Have a look at log files, like in step 7, and you will see which worker handled each task.
+
+## Monitor your Celery Cluster
+
+The `celery` binary provide some commands to monitor workers and tasks, far more convenient than browsing log files:
+
+1.  Use the **status** command to get the list of workers:
+
+        celery -A downloaderApp status
+
+    {{< output >}}
+worker1@celery: OK
+worker2@celery: OK
+celery@celery: OK
+{{< /output >}}
+
+2.  Use the **inspect active** command to see what the workers are currently doing:
+
+        celery -A downloaderApp inspect active
+
+    {{< output >}}
+-> worker1@celery: OK
+    - empty -
+-> worker2@celery: OK
+    - empty -
+-> celery@celery: OK
+    - empty -
+{{< /output >}}
+
+3.  Use the **inspect stats** command to get statistics about the workers. It gives lot of informations, like worker ressource usage under `rusage` key, or the total tasks completed under `total` key.
+
+        celery -A downloaderApp inspect stats
+
+
+## Monitor your Celery cluster with Flower
+
+Flower is a web-based monitoring tool that can be used instead of the `celery` command.
```

Comment:
@Jskobos do we want to include a "why" here? Something like, "Use Flower because it smells nicer than celery"

otherwise, we're just suggesting people use this instead of that 1:1, without reason

Reply:
It uses a web browser, so it's more colorful than the CLI. Could be worth adding that bit instead of the 'the celery command' clause.

Also the output tags won't format right quite yet but please leave them as is. I'm going to try to merge in the new output shortcode today or Mon morning.

Reply:
[STET]

looked at this again, within context - disregard my comment. It's a UI and it's nice looking.